acceptancetests/jujupy/workloads.py

```diff
@@ -0,0 +1,141 @@
+# This file is part of JujuPy, a library for driving the Juju CLI.
```

Comment:
This is a nice addition. Can we clean up the methods to standardize the calls in anyway? I'm thinking about reuse and ensuring the method calls make sense. Essentially test authors can deploy X software, and then they will want to validate X software. Would they ever just want to deploy and not validate? I can definitely see reasons to call validate again (after model changes, like upgrades).